manim/mobject/graph.py

```diff
@@ -212,6 +179,55 @@ def slide(v, dx):
     return {v: (np.array([x, y, 0]) - center) * sf for v, (x, y) in pos.items()}
 
 
+_layouts = {
+    "circular": nx.layout.circular_layout,
+    "kamada_kawai": nx.layout.kamada_kawai_layout,
+    "partite": _partite_layout,
+    "planar": nx.layout.planar_layout,
+    "random": _random_layout,
+    "shell": nx.layout.shell_layout,
+    "spectral": nx.layout.spectral_layout,
+    "spiral": nx.layout.spiral_layout,
+    "spring": nx.layout.spring_layout,
+    "tree": _tree_layout,
+}
```

Comment:
It might be useful to add a type in typing.py with `Literal` such that you can type the `str` arguments and get autocompletion for which arguments are available